src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ReconfigureFundingCycleSettingsPage/FundingAttributes.tsx

```diff
@@ -308,6 +308,28 @@ export function AllowSetTerminalsStatistic({
   )
 }
 
+export function PauseTransfersStatistic({
+  pauseTransfers,
+}: {
+  pauseTransfers: boolean
+}) {
+  return (
+    <Statistic
+      title={
+        <TooltipLabel
+          label={t`Pause transfers`}
+          tip={
+            pauseTransfers
+              ? t`Transfers are paused`
```

Comment:
"Project token transfers are paused/unpaused. This does not apply to ERC-20 tokens if issued."